test/performance/monitoring/node_exporter/node-exporter.yml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: node-exporter
+  namespace: kube-system
+  labels:
+    app: node-exporter
+spec:
+  selector:
+    matchLabels:
+      app: node-exporter
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: node-exporter
+    spec:
+      containers:
+      - image: quay.io/prometheus/node-exporter:latest
```

Comment:
Better to indicate the image version of third-party components to avoid the compatibility issue in the future.